index.js

```diff
@@ -25,11 +36,19 @@ module.exports = promisedHandlebars
  *   the promised results. This placeholder may not occur in the template or any partial. Neither
  *   my any helper generate the placeholder in to the result. Errors or wrong replacement will
  *   happen otherwise.
+ * @param {Promise} options.Promise the promise implementation to use. Defaults to global.Promise
  * @returns {Handlebars} a modified Handlebars object
  */
 function promisedHandlebars (Handlebars, options) {
   options = options || {}
   options.placeholder = options.placeholder || '\u0001'
+  if (options.Promise) {
```

Comment:
Mhm, I've thought about it and maybe breaking backward-compatibillity and going for 2.0 is the better option. The reason why I wanted backward compatibilty is that I don't like the idea of skipping support for old versions. There are some libraries out there that have major releases every couple of month. That's fine. But I also would like to have bugfixes for an old version for at least a year or so. `promised-handlebars` is not so far yet, so let's just break backward compatibillity. I can still provide bugfixes for 1.0.x later. There just won't be a 1.1.

So just keep it like this

```js
var Promise = options.Promise || global.Promise
if (!Promise) {
  throw new Error(...)
}
```